src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateService.kt

```diff
@@ -25,18 +25,15 @@ class ReleaseDateService(
 
   fun isInHardStopPeriod(licence: Licence, overrideClock: Clock? = null): Boolean {
     val now = overrideClock ?: clock
-    val endOfHardStopPeriod = getCutOffDateForLicenceTimeOut(now)
-    val releaseDate = licence.actualReleaseDate ?: licence.conditionalReleaseDate
+    val hardStopDate = getHardStopDate(licence)
+    val today = LocalDate.now(now)
 
-    if (releaseDate == null) {
+    if (hardStopDate == null) {
       log.warn("Licence with id: ${licence.id} has no CRD or ARD")
       return false
     }
-    return if (releaseDate < LocalDate.now(now)) {
-      false
-    } else {
-      releaseDate <= endOfHardStopPeriod
-    }
+
+    return today >= hardStopDate && today <= licence.licenceStartDate
```

Comment:
Discussed with Seb, using licenceStartDate is doing ARD || CRD and also means that this logic would be false for early releases.